TheMET/TheMET/DataModels/Art.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Art.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 21/03/2023.
+//
+
+import Foundation
+
+class Art: Decodable {
+    
+    let objectID: Int
+    
+    let primaryImage: String?
```

Comment:
Let's name it as `primaryImageURLString: String?`

Reply:
Fixed